cosmo_tester/framework/test_hosts.py

```diff
@@ -856,8 +856,16 @@ def _save_manager_logs(self):
             'Attempting to save manager logs for test:  {0}'.format(test_path))
         logs_dir = os.path.join(os.path.expanduser(logs_dir), test_path)
         util.mkdirs(logs_dir)
-        for instance in self.instances:
-            self._save_logs_for_instance(instance, logs_dir)
+        for i, instance in enumerate(self.instances):
+            instance_deleted = getattr(instance, "deleted", None)
+            self._logger.debug('Instance {0} deleted status: {1}'.format(
+                i, instance_deleted))
+            if instance_deleted is None or instance_deleted:
```

Comment:
The two terms are equal in this case

Reply:
Wanted to make it extra explicit, I'll change it